packages/mx-adapter/src/adapter.ts

```diff
@@ -190,7 +277,7 @@ export class MxAdapter implements WidgetAdapter {
         credentials: request.credentials?.map(
           (credential) =>
             ({
-              guid: credential.id,
+              guid: credential.guid || credential.id,
```

Comment:
This update allows me to call the `UpdateConnection` from `CreateConnection` without having to revert the credentials on `newMemberProperties` back to `id` and `value` after it's already been changed to guid in the `GetNewMemberPropertiesFromRequest` function.